actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -111,7 +111,7 @@ def fetch_or_cache_partial(cached_partials, template, order_by:)
         end
 
         unless entries_to_write.empty?
-          collection_cache.write_multi(entries_to_write)
+          collection_cache.write_multi(entries_to_write, @options.slice(:expires_in))
```

Comment:
Are there other caching options that could be useful here?

Reply:
I don't know if there are, but nothing stood out to me.

Reply:
I think `expires_at` would be useful too.